sprint0/sprint0/Collision/EnemyEnemyCollisionHandler.cs

```diff
@@ -11,23 +11,29 @@ public EnemyEnemyCollisionHandler()
 
         public void HandleCollision(IEnemy enemy1, IEnemy enemy2, Direction side)
         {
-            switch (side)
+           if ((enemy1 is ManhandlaLimb && enemy2 is Manhandla)|| (enemy2 is ManhandlaLimb  && enemy1 is Manhandla)
+               ) { }
```

Comment:
I think it would be preferable to have the not version of this and 1 if block instead of an if else where the if block is empty